.gitlab-ci.yml

```diff
@@ -856,6 +856,11 @@ library:ci-http:
   - library:ci-menhir
   - library:ci-itree_io
   - plugin:ci-quickchick
+library:ci-trakt:
```

Comment:
```suggestion

library:ci-trakt:
```

Reply:
I committed the suggested change but I think you suggested to remove precisely this whitespace in a previous commit, did I miss something?

Reply:
The previous commit had trailing whitespace.
Possibly the github UI is bugged, it showed
~~~diff
-  
~~~
but should have been
~~~diff
-  
+
~~~